sdks/js/packages/core/react/contexts/FrontierContext.tsx

```diff
@@ -274,8 +279,10 @@ export const FrontierContextProvider = ({
           orgId,
           billingId
         );
-        if (resp?.data?.subscriptions?.length) {
-          const activeSub = getActiveSubscription(resp?.data?.subscriptions);
+        const subscriptionsList = resp?.data?.subscriptions || [];
+        setSubscriptions(subscriptionsList);
```

Comment:
What does `setSubscriptions` do?

Reply:
we use to keep only 1 active subscription in the state. after trial user can have 2 subscription running.
one is in active state and other is in trialing state.
and we already fetch all subscriptions in order to set active one.
so instead of fetching all subscriptions again for trial, i just save this subscriptions list in the state.